### PR TITLE
Resolve warnings in meraki mr radio

### PR DESCRIPTION
**Pull request by y0rune**

### CODE REVIEW

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml [hunk 1]
```diff
@@ -56,11 +57,12 @@
...
     register: create
 
-  - set_fact:
+  - name: Set fact create
+    ansible.builtin.set_fact:
       profile_id: "{{ create.data.id }}"
 
   - name: Configure AP radios in check mode
-    meraki_mr_radio:
+    ansible.cisco.meraki_mr_radio:
```
Comment (kbreit) [inline tests/integration/targets/meraki_mr_radio/tasks/main.yml:65]: This FQCN is incorrect. Please revise.

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml [hunk 2]
```diff
@@ -78,25 +80,27 @@
...
 
-  - assert:
+  - name: Assert query_one
+    ansible.builtin.assert:
       that:
         - query_one.data is defined
 
   - name: Configure AP radios
-    meraki_mr_radio:
+    ansible.cisco.meraki_mr_radio:
```
Comment (kbreit) [inline tests/integration/targets/meraki_mr_radio/tasks/main.yml:103]: Same with this one. Incorrect FQCN.

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml [hunk 3]
```diff
@@ -78,25 +80,27 @@
...
-  - assert:
+  - name: Assert create_check
+    ansible.builtin.assert:
       that:
         - create_check.data is defined
         - create_check is changed
 
   - name: Query a device radio
-    meraki_mr_radio:
+    ansible.cisco.meraki_mr_radio:
```
Comment (kbreit) [inline tests/integration/targets/meraki_mr_radio/tasks/main.yml:90]: Incorrect FQCN

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml [hunk 4]
```diff
@@ -113,13 +117,14 @@
...
-  - assert:
+  - name: Assert update
+    ansible.builtin.assert:
       that:
         - update.data is defined
         - update is changed
 
   - name: Configure AP radios
-    meraki_mr_radio:
+    ansible.cisco.meraki_mr_radio:
```
Comment (kbreit) [inline tests/integration/targets/meraki_mr_radio/tasks/main.yml:127]: Incorrect FQCN

**Review by kbreit — CHANGES_REQUESTED** [review]

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml (hunk 2 above)
Comment (y0rune) [reply tests/integration/targets/meraki_mr_radio/tasks/main.yml]: Okay! I did it. Sorry for my error.

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml (hunk 3 above)
Comment (y0rune) [reply tests/integration/targets/meraki_mr_radio/tasks/main.yml]: Okay! I did it. Sorry for my error.

File: tests/integration/targets/meraki_mr_radio/tasks/main.yml (hunk 4 above)
Comment (y0rune) [reply tests/integration/targets/meraki_mr_radio/tasks/main.yml]: Okay! I did it. Sorry for my error.

**Comment by kbreit** [comment]
Thank you for your contribution. Merging.

**Pull request merged**